sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -423,7 +504,9 @@ public ServiceBusReceiverClientBuilder isAutoComplete(boolean autoComplete) {
         /**
          * Sets if lock should be automatically renewed.
          *
-         * @param isLockAutoRenewed {@code true} if the lock should be automatically renewed; {@code false} otherwise.
+         * @param isLockAutoRenewed {@code true} if the lock should be automatically renewed; {@code false}
+         *     otherwise.
```

Comment:
This should probably also clarify that this will be done for Async client only. And will not autorenew for sync client.

Reply:
Fixed.